demos/bubblewrap/actors/bubble.py

```diff
@@ -1,10 +1,9 @@
 import numpy as np
 from queue import Empty
 import os
-
 import improv.store as store
-
-from bubblewrap import Bubblewrap
+from scipy import io as sio
+from Bubblewrap.bubblewrap import Bubblewrap
```

Comment:
Can we wrap this in a `try`, `except` block that emits an error when `Bubblewrap` is not installed (and so can't be imported? Since this and `proSVD` are not improv dependencies, we can use a `ModuleImportError` exception to print a message that tells users they need to install these libraries (and give the URL with install instructions). Eventually, that will just be `pip install proSVD`, etc., but for now, it can point to the GitHub main page.

Reply:
Currently, when I try to run the demo without these dependencies, it fails with the same (unhelpful) error in #129.